favorite_files.py

```diff
@@ -394,9 +425,18 @@ def check_st_version():
             window.run_command('open_file', {"file": "${packages}/FavoriteFiles/messages/upgrade-st-3080.md"})
 
 
+def settings():
+    return sublime.load_settings("favorite_files.sublime-settings")
+
+
 def plugin_loaded():
     """Setup plugin."""
+    global Favs, global_favs_file
```

Comment:
I don't see a need for `global_favs_file`.  Can't this just be acquired from the Favs object?  Why a new dedicated global?